tests/routes/visitor_check.test.js

```diff
@@ -45,7 +45,7 @@ test('POST /api/visit/check | viable name & email & not found in database', t =>
     .post('/api/visit/check')
     .set('authorization', token)
     .send(successPayload)
-    .expect(400)
+    .expect(200)
```

Comment:
`200` feels a bit wrong here..

Reply:
Why? The purpose of the endpoint is to validate whether the payload is valid for creating a new visitor record. If no existing visitor record with that email exists, it is valid.

Reply:
Ahhh I didn't notice you'd swapped the ternary in the route. This test makes more sense now.